libraries/freertos_plus/standard/https/include/iot_https_client.h

```diff
@@ -344,6 +409,131 @@ IotHttpsReturnCode_t IotHttpsClient_SendSync(IotHttpsConnectionHandle_t connHand
+ * 
+ * This function will invoke each of the non-NULL callbacks configured in #IotHttpsAsyncInfo_t.callbacks when the
+ * scheduled asynchronous request is progress. Please see #IotHttpsClientCallbacks_t for information on each of the 
+ * callbacks.
+ * 
+ * After this API is executed, the scheduled async response will store the response headers as received from 
+ * the network, in the header buffer space configured in #IotHttpsResponseInfo_t.userBuffer. If the 
+ * configured #IotHttpsResponseInfo_t.userBuffer is too small to fit the headers received, then headers that don't 
+ * fit will be thrown away. Please see #responseUserBufferMinimumSize for information about sizing the 
+ * #IotHttpsResponseInfo_t.userBuffer.
+ * 
+ * If #IotHttpsRequestInfo_t.isNonPersistent is set to true, then the connection will disconnect, close, and clean all 
+ * taken resources automatically after receiving the first response.
+ * 
+ * See @ref connectionUserBufferMinimumSize for information about the user buffer configured in 
+ * #IotHttpsConnectionInfo_t.userBuffer needed to create a valid connection handle.
+ * 
+ * @param[in] connHandle - Handle from an HTTPS connection.
+ * @param[in] reqHandle - Handle from a request created with IotHttpsClient_initialize_request.
+ * @param[out] pRespHandle - HTTPS response handle.
+ * @param[in] pRespInfo - HTTP response configuration information.
+ * 
+ * @return One of the following:
+ * - #IOT_HTTPS_OK if the request was sent and the response was received successfully.
+ * - #IOT_HTTPS_MESSAGE_TOO_LARGE if the response cannot fit in the configured 
+ *      IotHttpsRequestHandle_t.response_message.headers and IotHttpsRequestHandle_t.response_message.body.
+ * - #IOT_HTTPS_CONNECTION_ERROR if the connection failed.
+ * - #IOT_HTTPS_FATAL if there was a grave error with the last async job finishing.
+ * - #IOT_HTTPS_ASYNC_SCHEDULING_ERROR if there was an error scheduling the asynchronous request.
+ * - #IOT_HTTPS_INTERNAL_ERROR if there was an internal error with starting an asyncrhonous request servicing task.
+ * - #IOT_HTTPS_INVALID_PARAMETER if there were NULL parameters or the request passed in was a synchronous type.
+ *          
+ */
+/* @[declare_https_client_sendasync] */
+IotHttpsReturnCode_t IotHttpsClient_SendAsync(IotHttpsConnectionHandle_t connHandle, 
+                                              IotHttpsRequestHandle_t reqHandle, 
+                                              IotHttpsResponseHandle_t * pRespHandle, 
+                                              IotHttpsResponseInfo_t* pRespInfo);
+/* @[declare_https_client_sendasync] */
+
+/**
+ * @brief Cancel an Asynchronous request.
+ * 
+ * This will stop an asynchronous request.
+ * 
+ * If this is called before the scheduled asynchronous request actually runs, then request will not be sent. 
+ * If this is called during any of the asynchronous callbacks, then the request/response will stop processing when the 
```

Comment:
If this is called during any of the asynchronous callbacks, then the library stops processing the request/response when the callback returns.

Reply:
Thanks.